dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -253,8 +253,10 @@ public void findOneTest() throws Exception {
         Matcher<? super Object> publicItem1Matcher = ItemMatcher.matchItemWithTitleAndDateIssued(publicItem1,
                         "Public item 1", "2017-10-17");
 
+        String token = getAuthToken(admin.getEmail(), password);
```

Comment:
See docs here: https://github.com/atmire/DSpace/blob/w2p-69349_edit-community-assign-roles/dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java#L257

Reply:
if not necessary please revert it otherwise copy the original test in an ignored test and report about it in a jira ticket (it could become a test to demonstrate the resolution of https://jira.lyrasis.org/browse/DS-4444 )